torch/autograd/gradcheck.py

```diff
@@ -162,10 +162,13 @@ def get_numerical_jacobian(fn, inputs, target=None, eps=1e-3, grad_out=1.0):
                   "in a future version of PyTorch. If you have a specific use for "
                   "this or feature request for this to be a stable API, please file "
                   "us an issue at https://github.com/pytorch/pytorch/issues/new")
+    if grad_out != 1.0:
```

Comment:
nit: could you add a comment here saying that the parameter is only kept for BC reasons